### ISSUE TITLE
Necronomicon text: the redraftening!

### ISSUE THREAD

**Issue by Shinoow** [Shinoow/AbyssalCraft#264]
Since I've been sloppy with keeping the Necronomicon up-to-date with all stuff that has changed between updates, it's gotten behind on relevant and accurate information in some sections. I did a development livestream earlier this week ([Shameless plug](https://www.twitch.tv/shinoow)) where I started to redraft some of the text (mainly focused on the progression sections for each dimension, and a little in the other subsections there).
Since there are people who actively use AbyssalCraft (or do stuff with it from time to time) I thought I'd open a dialog so I can get more opinions on things that could use some freshening up (or just straight up be rewritten due to the information being grossly outdated).

**Comment by Shinoow** [comment]
Yeah, a new thing introduced in 1.9.4.4 (which will be documented in 1.9.4.5 once I get that update out) is that items and blocks are partially locked behind obtaining knowledge (currently it's just the name getting obscured, but there is room for other minor limitations until the required knowledge has been obtained). The whole process of obtaining knowledge right now is just a matter of exploring biomes/dimensions, and killing mobs.

**Comment by P3rf3ctXZer0** [comment]
Love it ;) )

**Comment by Levitikan** [comment]
if you have already got an abyssalnomicon on a world and the update blocks all the text is there a way to unlock it without having to do everything again?

**Comment by Mochnant** [comment]
I could not find anything in the book to explain how to get PE, or how to set up rituals.  I had to check wikis to figure that out, and even then it was not super clear.

**Comment by Shinoow** [comment]
Did you check the section named `Rituals`? That's where that information is at.

**Comment by Mochnant** [comment]
I looked through the book several times, to no avail.  I have to assume that section was not unlocked for me at the time (or operator error!).  What is the trigger to reveal it?

**Comment by Shinoow** [comment]
There's no trigger to reveal it. There's a button for it at the front page.

**Comment by Mochnant** [comment]
It was user error, thanks.  That said, the navigation UI could use some improvement.  It is easy to get lost in the book.  A sidebar index, perhaps?

**Comment by Shinoow** [comment]
Yeah, I could look into that.

**Comment by Mike-U5** [comment]
One thing that bugs me mildly is that when you pick "The Great Old Ones" in the Forbidden Knowledge section, you get another page where you can pick between "The Great Old Ones" again and "The Outer Gods". Seems a bit redundant, maybe just put both of those separately in the Forbidden Knowledge section.

So it'd look like this:
![edit](https://user-images.githubusercontent.com/32613991/38778810-5378dc54-40bf-11e8-9be3-62a755bddb3f.png)

**Comment by Masterwills** [comment]
I could give you the idea of ​​starting a second part of the abissalcraft because the end has more to give, a second part where you have spent a lot of time in which you have to explore the four dimensions and discover what happened before, new bosses and the return of j'hazar with more force than ever new weapons creatures mysteries and an area in the center of the dimensions that is called the nexus area through that sona podras you can access other dimensions and return to dimensions that you visited before from that point I think the idea is good send me a message if you are interested in the idea my facebook is masterwills lanzasñ the ñ is so that it does not translate the word

**Comment by Hypsellis** [comment]
You could maybe use the new api Patchouli developped by Vazkii which enables to create really good guide books. It seems that most of big mods will write their guide with this api when they will update to the 1.14.3.
